contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-react-native.mdx

```diff
@@ -76,6 +76,27 @@ Or when you want to trigger the reload, but don't care about the result:
 posthog.reloadFeatureFlags()
 ```
 
+### Feature flag caching
+
+> **Warning:** The React Native SDK caches feature flag values in AsyncStorage. Cached values persist indefinitely with no TTL until updated by a successful API call. This enables offline support and reduces latency, but means **inactive users may see stale flag values** from their last session.
```

Comment:
I think it make more sense if we just let this be a normal paragraph, that way the second sentence makes more sense. 

```suggestion
The React Native SDK caches feature flag values in AsyncStorage. Cached values persist indefinitely with no TTL until updated by a successful API call. This enables offline support and reduces latency, but means **inactive users may see stale flag values** from their last session.
```